src/mmrelay/matrix_utils.py

```
@@ -1802,14 +1833,7 @@ async def join_matrix_room(matrix_client, room_id_or_alias: str) -> None:
     if room_id_or_alias.startswith("#"):
         try:
             response = await matrix_client.room_resolve_alias(room_id_or_alias)
-        except (
-            NioLocalProtocolError,
-            NioRemoteProtocolError,
-            NioErrorResponse,
-            NioLocalTransportError,
-            NioRemoteTransportError,
-            asyncio.TimeoutError,
-        ):
+        except Exception as e:
```

Ruff (0.14.5)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)